src/runtime/hexagon_remote/remote.cpp

```diff
@@ -21,6 +21,57 @@ extern "C" {
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+#define MAX_WORKER_THREADS 4
+#define NUM_WORKER_THREADS_TO_CREATE 3
+#define STACK_SIZE 4096
```

Comment:
How flexible is this stack size? We like to use the stack where possible for allocations to avoid the cost of malloc, 4 kb isn't much...